sec-policy/selinux-base-policy/selinux-base-policy-2.20200818-r2.ebuild

```diff
@@ -39,6 +39,7 @@ PATCHES=(
 	"${FILESDIR}/init.patch"
 	"${FILESDIR}/locallogin.patch"
 	"${FILESDIR}/logging.patch"
+	"${FILESDIR}/unlabeled.patch"
```

Comment:
Maybe we can annotate here that this was for torcx and it could be dropped in the future

Reply:
Good idea and let's add a reference in: https://github.com/flatcar-linux/Flatcar/issues/443